.circleci/build.sh

```diff
@@ -24,24 +24,36 @@ install_pocl() {
    cd ../
 }
 
-# Install Glow dependencies
-sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main"
-sudo apt-get update
-sudo apt-get install -y llvm-8 clang-8 llvm-8-dev libpng-dev libgoogle-glog-dev
+if [ "${CIRCLE_JOB}" != "CHECK_CLANG_FORMAT" ]; then
+    # Install Glow dependencies
+    sudo apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-8 main"
+    sudo apt-get update
+    sudo apt-get install -y llvm-8 clang-8 llvm-8-dev libpng-dev libgoogle-glog-dev
 
-# Redirect clang
-sudo ln -s /usr/bin/clang-8 /usr/bin/clang
-sudo ln -s /usr/bin/clang++-8 /usr/bin/clang++
-sudo ln -s /usr/bin/llvm-symbolizer-8 /usr/bin/llvm-symbolizer
-sudo ln -s /usr/bin/llvm-config-8 /usr/bin/llvm-config-8.0
-
-GLOW_DIR=$PWD
+    # Redirect clang
+    sudo ln -s /usr/bin/clang-8 /usr/bin/clang
+    sudo ln -s /usr/bin/clang++-8 /usr/bin/clang++
+    sudo ln -s /usr/bin/llvm-symbolizer-8 /usr/bin/llvm-symbolizer
+    sudo ln -s /usr/bin/llvm-config-8 /usr/bin/llvm-config-8.0
+else
+    sudo -E apt-add-repository -y "ppa:ubuntu-toolchain-r/test"
```

Comment:
do we need to do all these things to install llvm-7? (that's probably just an artifact of this being run on travis and now we can get rid of this).
Just do similar things as you do above, but for llvm-7.